docs/tutorial.md

```diff
@@ -187,20 +188,58 @@ func halfRoundedUp(n int) (res int) {
 ```
 
 Currently, the `pure` modifier has to be written after the pre and postcondition and before the `func` keyword. Furthermore, pure functions and methods have strict syntax constrains, the body must be a single return statement that returns a single deterministic side-effect free expression.
+Furthermore, `pure` functions must terminate. Thus, we add a `decreases` keyword instructing Gobra to prove termination as explained next.
+
+
+## Total Correctness
+
+By default, Gobra proves partial correctness, i.e., if a function terminates then its postcondition provably holds.
+To prove total correctness, a so-called termination measure must be provided.
+A termination measure is an assertion that is bounded from below and decreases in each iteration or recursive call, allowing Gobra to prove that a function or loop provably termination.
```

Comment:
hmm, I wouldn't call it an assertion (many assertions are not valid termination measures, like `A --* B` and `acc(x.f)`) and termination measures may not be assertions (e.g., constant `0`).
I'd rather say an expression that evaluates to a type for which there is a well-founded order